migrations/migrations.go

```diff
@@ -6,8 +6,6 @@ package migrations
 import (
```

Comment:
Another quick note on this: can we move this package under `jobs` like we have done for the other jobs? Now with the new migration system, we also have db/migrations which can confuse users with a top-level package like this.

Reply:
Sure, but I think we can do that in another PR, this is already big enough.

Reply:
I finally moved it to the jobs directory, was pretty straight forward.